integration-tests/MultichainApp.tsx

```diff
@@ -15,7 +15,7 @@ export function MultichainApp() {
   return (
     <Provider store={store}>
       <Updater chainId={ChainId.MAINNET} blockNumber={blockNumberMainnet} />
-      <Updater chainId={ChainId.RINKEBY} blockNumber={blockNumberRinkeby} />
+      <Updater chainId={ChainId.RINKEBY} blockNumber={blockNumberRinkeby} blocksPerFetch={5} />
       <Home chainIds={chains} blockNumbers={blocks} />
```

Comment:
Why rinkeby, but not mainnet here?

Reply:
Just testing that we can set blocksPerFetch to different values for different chains. I presume ETH will usually default to blocksPerFetch = 1 and other chains will be configured differently so this just simulates that.